src/main/perl/Path.pm

```diff
@@ -214,17 +216,20 @@ sub _safe_eval
     %opts = %$optsref if $optsref;
 
     local $@;
-    eval {
-        $res = $funcref->(@args, %opts);
-    };
+    my $status = eval {
```

Comment:
Not exactly. This is `$res` if the evaluated function succeeds. But it is also `undef` in case the evaluated function has a syntax or runtime error. in this case I agree that `$res` being undefined, the result should be the same.